### ISSUE TITLE
[a11y]: ProgressStep component reporting violation of `2.5.8 Minimum Target Size`

### ISSUE THREAD

**Issue by nkn2022** [carbon-design-system/carbon#17352]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.57.0

### React version

17.0.2

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker

### Assistive technology

No

### Description

https://react.carbondesignsystem.com/?path=/story/components-progressindicator--playground&args=vertical:!true

Run the tool with 7.3 requirement. 
See the violation on 
`<span class="cds--progress-line">`

<img width="534" alt="image" src="https://github.com/user-attachments/assets/226e8304-c796-4c42-ab2f-a1f90c3b0673">

[IBM 2.5.8 Target Size (Minimum)](https://www.ibm.com/able/requirements/requirements/?version=v7_3#2_5_8)
Undersized target "span" does not have sufficient spacing of 12 CSS pixels from another target "button"

### WCAG 2.1 Violation

IBM 2.5.8 Target Size (Minimum)

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-progressindicator--playground&args=vertical:!true

### Steps to reproduce

Run the tool with IBM Equal Access Accessibility Checker (7.3)

### Suggested Severity

None

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by alisonjoseph** [comment]
This is most likely a false positive, there should be an update to the ruleset from the IBMa team soon.

**Comment by 2nikhiltom** [comment]
Hey @nkn2022 ! I do not see this issue on the latest deployment, I am missing something ?
Could you confirm this on the latest version of checker 

![Image](https://github.com/user-attachments/assets/9891f6eb-10be-4eab-aa89-a044952e01a9)

**Comment by nkn2022** [comment]
@2nikhiltom Looks like the accessibility checker tool itself was updated for some corrections. I do not see this issue reported anymore on our UI using the same version of carbon react :) 

 ![Image](https://github.com/user-attachments/assets/8975e516-d9cc-4666-8043-256bf5e563ae)


Will close this as not reproducible anymore. Thanks for checking.
